### DISCUSSION TITLE
Virtual reality or mixed reality

### DISCUSSION THREAD

**Discussion by Addinho** [f1multiviewer/issue-tracker#524] [Feature requests] [answered]
Assuming quality of VR device is good enought it would be cool to view racing in VR or mixed reality with lots of space for many "screens". I quess it is in some way possible through Virtual Desktop or similar solutions for Meta Quest or Vision Pro. A native app for the device might give better immersion but might suffer in performance when running many screens.

**Comment by LapsTimeOFF** [comment, marked as answer]
We do not plan on supporting platforms other than Windows, macOS and Linux as this would be complex to do because of DRM.
